lib/android/app/src/main/java/com/reactnativenavigation/anim/TopBarAnimator.java

```diff
@@ -62,7 +72,9 @@ private AnimatorSet getDefaultShowAnimator(float startTranslation, TimeInterpola
     }
 
     public void hide(AnimationOptions options, AnimationListener listener) {
-        if (options.hasValue()) {
+        if (options.hasValue() && options.id.hasValue() && options.id.get().equals(stackId)) {
+            hideAnimator = options.getAnimation(topBar);
```

Comment:
line 76 is the same as line 78, is this conditional required?

Reply:
Same here